core/storage/sqlite3_ondisk.rs

```diff
@@ -248,7 +248,7 @@ pub enum PageType {
 }
 
 impl TryFrom<u8> for PageType {
-    type Error = crate::error::LimboError;
+    type Error = LimboError;
```

Comment:
In the future, please resist the urge to mix random cleanups with a pull request. They can make pull requests harder to review and merge.

Reply:
Sure!